src/SourceGenerators/Uno.UI.SourceGenerators/XamlGenerator/XamlCodeGenerator.cs

```diff
@@ -29,9 +29,9 @@ public void Execute(GeneratorExecutionContext context)
 
 			// No initialization required for this one
 			//if (!Process.GetCurrentProcess().ProcessName.Equals("devenv", StringComparison.OrdinalIgnoreCase))
-			{
-				// Debugger.Launch();
-			}
+			//{
```

Comment:
Codacy found an issue: [Remove this commented out code.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=6377855)